pkg/lib/config/configsource/database.go

```diff
@@ -52,7 +52,20 @@ func (d *Database) ResolveAppID(r *http.Request) (string, error) {
 		host = h
 	}
 
-	return "", nil
+	var appID string
+	err := d.Database.WithTx(func() error {
+		aid, err := d.Store.GetAppIDByDomain(host)
```

Comment:
We should be able to resolve default domain even if they are not in the `domain` table.

Reply:
@louischan-oursky I found that we stored both default and custom domain in the `domain` table, so would suggest to resolve the app id in the same way. What do you think?

Reply:
If we store the default domain in the table, then we cannot simply change a env var to change every default domain of all apps.

Reply:
Since the domain is written in `authgear.yaml` `public_origin` field, even we support resolve default domain from env var, we cannot change it. After discussion, we will resolve both default and custom domain from db for consistent.